Description: Upstream changes introduced in version 0.1-0ubuntu0~ppa2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 invertika (0.1-0ubuntu0~ppa2) lucid; urgency=low
 .
   * Initial release
 .
 The person named in the Author field signed this changelog entry.
Author: Jonathan Kolberg <bulldog98@kubuntu-de.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- invertika-0.1.orig/invertika.desktop
+++ invertika-0.1/invertika.desktop
@@ -8,7 +8,7 @@ Comment[de]=ein deutsches Online Fantasy
 Comment[it]=Un gioco fantasy online
 Comment[nl]=Een online role playing game
 Comment[sv]=Ett fantasyrollspel online
-Exec=invertika /usr/local/share/invertika/branding.xml
+Exec=invertika /usr/share/invertika/branding.xml
 StartupNotify=false
 Terminal=false
 Type=Application
--- invertika-0.1.orig/invertika.menu
+++ invertika-0.1/invertika.menu
@@ -1,4 +1,4 @@
 ?package(mana):needs="X11" section="Games/Adventure"\
  title="Invertika"\
  description="Invertika is a 2D MMORPG client."\
- command="/usr/games/invertika-client" icon="/usr/share/pixmaps/ivk.xpm"
+ command="/usr/games/invertika /usr/share/invertika/data/branding.xml" icon="/usr/share/pixmaps/ivk.xpm"
